src/index.html

```diff
@@ -22,9 +22,10 @@ <h2>Recent Scores</h2>
       <section class="add-score flex">
         <h2 class="add-score">Add your score</h2>
         <form id="scoreForm" class="add-score flex" action="">
-          <input type="text" name="name" placeholder="Your name" />
-          <input type="text" name="score" placeholder="Your score" />
+          <input type="text" name="name" placeholder="Your name" required />
+          <input type="text" name="score" placeholder="Your score" req />
```

Comment:
- [Optional] Please make sure the type for the score is `number` to make sure all the data the user can enter is a number.